config/crds/clusterlink.net_imports.yaml

```diff
@@ -40,6 +40,10 @@ spec:
           spec:
             description: Spec represents the attributes of the imported service.
             properties:
+              dnsName:
+                description: DnsName is an optional external dns name for this imported
```

Comment:
Based on Go's convention this should be `DNSName` (should be changed in the Go struct and then generated to here).
The YAML field can still be `dnsName`